src/domain/organisation/contextProviders/OrganisationStoragePersistor.tsx

```diff
@@ -0,0 +1,58 @@
+import { organisationLocalStorageKey } from '../constants';
+
+/**
+ * Provides utility methods for saving, loading, and clearing the active organisation ID from local storage.
+ */
+export default class OrganisationStoragePersistor {
+  /**
+   * Saves the provided organisation ID to local storage.
+   *
+   * @param organisationId - The ID of the organisation to save.
+   * @example
+   * OrganisationStoragePersistor.save('12345');
+   */
+  static save(organisationId: string): void {
+    // eslint-disable-next-line no-console
+    console.debug('OrganisationStoragePersistor:save', organisationId);
+    localStorage.setItem(
+      organisationLocalStorageKey,
+      JSON.stringify(organisationId)
+    );
```

Comment:
Wouldn't it be good to prefix the storage key with the profile ID / user ID here, as now the same key is shared for multiple users?

Reply:
I don't think it gives any value. The OIDC key itself is not persisted like that either. Also, the old implentation was not done like that. I think it would make things more complicated, but does not bring anything more. When a new login is made, the organisation selection is "random", since it just uses the first organistaion in a list of organisations, which are in random order. If the organisation is picked from local storage, it is still validated against the organisations that are set for profile. Also, the last validation should be done by the API, whether or not the client has permissions to manage the organisation.

Reply:
Actually, I even started thinking that I could remove the environment and app specific key too, since the organisation id will be unique in API anyway, so it cannot match in different apps and envs. We should not collect a list of organisations in local storage of a browser.

Reply:
Why remove the app specific key? For sure the organisation ID is related to this application and not some other, and just to make it clear that it doesn't possibly overlap with some other app's data it'd be justified to use at least an app specific key here.